sympy/core/sympify.py

```diff
@@ -435,14 +435,14 @@ def sympify(a, locals=None, convert_xor=True, strict=False, rational=False,
     if iterable(a):
         try:
             return type(a)([sympify(x, locals=locals, convert_xor=convert_xor,
-                rational=rational) for x in a])
+                rational=rational, evaluate=evaluate) for x in a])
         except TypeError:
             # Not all iterables are rebuildable with their type.
             pass
     if isinstance(a, dict):
```

Comment:
The case of `dict` is already handled -- as for tuple -- by a converter in `containers.py`. The evaluate keyword has no affect. This is dead code and can probably be deleted.

Reply:
So, evaluate flag from line 445 in `sympify.py` can be removed along with all the tests for dict items in `test_sympify.py`.